### ISSUE TITLE
Plot joint distribution?

### ISSUE THREAD

**Issue by ebachelet** [SebastianBocquet/pygtc#33]
Hello

Thanks for the nice package. This is not an issue, but a request. Is there, or would it be possible to add, an option to plot the joint distribution? For example, plot in a different color the joint distribution of Planck and WMAP?

Thanks

**Comment by SebastianBocquet** [comment]
Thanks for the feedback!

To make sure I understand: so you have two chains (e.g., Planck and WMAP), and you want to plot both of them (which is already possible) but you also want to plot the joint constraint?
In general, one could try and do this via importance sampling, but if the two distributions do not have much overlap this won't work well. So typically, it might be better to explicitly run the joint analysis. Is that an option?

**Comment by ebachelet** [comment]
Yes. So you recommend to generate manually the joint sampling and adding it as a third component? I will give it a try.

**Comment by SebastianBocquet** [comment]
Sounds good. Please don't hesitate to reach out with more questions!
